third-party/realdds/src/dds-device-server.cpp

```diff
@@ -176,6 +179,11 @@ void dds_device_server::init( std::vector< std::shared_ptr< dds_stream_server >
         else
             DDS_THROW( runtime_error, "unexpected stream '" + stream->name() + "' type" );
     }
+
+    // Create a control server and set callback
+    _control_server = std::make_shared< dds_control_server >( _subscriber, _topic_root + "/control" );

```

Comment:
Seems weird: why is the device server creating a control server? It's a control client -- the server is on the client side... i.e., the "server" is the "publisher", not the subscriber...

Reply:
The "Server" term refers to the camera side, as opposed to the "client" or "host".
"Server" does imply publisher. Many server only handles incoming requests.

Reply:
No, let's stick to one terminology: a server "serves" data (i.e., sends), a client receives. Otherwise it's confusing.

Reply:
I think that your terminology is confusing. A server just have services that handle client requests.

And what is the difference than between "broadcaster" and "server"? They both send data.

Reply:
I didn't make the terminology. A server serves data, whether it's web pages or anything else.
The "broadcaster" needed distinguishing in its name somehow, and we decided on broadcaster which you're right, also serves data. But it's understood from the name.
But here you have something that consumes data and serves nothing. Why call it a server?

Reply:
I don't agree with you here.
All the `-server` classes are on the "server" side (as in `rs-dds-server` tool) to distinguish from the "host" side.
`dds-device-server` vs `dds-device`, `dds-stream-server` vs `dds-stream`.
Keeping this notation helps to orient in the code and distinguish the class purpose.

@Nir-Az - what do you think?

Reply:
This argument might be redundant, as mentioned in another conversation this class might later be refactored out...

Reply:
I agree that `server` should not be confused and places at the client side.
`server` is the `node` that sends frames, it a camera server.
It will add more confusion to call the client server.

We are creating here a client for controlling the camera server and getting frames..
It does help distinguish between sides IMO.